packages/sdk/Dockerfile

```diff
@@ -105,6 +105,7 @@ COPY devnet /usr/local/bin
 COPY eth_isready /usr/local/bin
 COPY eth_dump /usr/local/bin
 COPY eth_load /usr/local/bin
+COPY create_image /usr/local/bin
```

Comment:
`create_image` can confuse with the ext2 image and others, at least for me.

Maybe `create_snapshot` or even `create_machine_snapshot`.